services/meta/association.md

```diff
@@ -1,45 +1,75 @@
+
+## Service Props
+
+Name             | Supported Values                                | Description                                                             |
+-----------------|-------------------------------------------------|-------------------------------------------------------------------------|
+`in_services`    | `["out_bin_switch", "out_lvl_switch", "color"]` | Zigbee only. List of services that can be controlled, e.g. on a lamp.   |
+`out_services`   | `["out_bin_switch", "out_lvl_switch", "color"]` | Zigbee only. List of services the device can control, e.g. on a button. |
+
+> A controllable device (like a lamp) would have a list of `in_services` that would indicate the services it supports. A controller (like a button) would have a list of `out_services` instead.
+
+## Definitions
+* `association_members` is an object describing association group members:
+ 
+| Field        | Type      | Example           | Description                                    |
+|--------------|-----------|-------------------|------------------------------------------------|
+| `group`      | int       | `1`, `3`, `5`     | Id of the supported group.                     |
+| `members`    | str_array | `["2_0", "15_1"]` | List of group members.                         |
+
+> The `group` field is completely ignored for zigbee. The field can be null, empty or removed completely for commands and will be missing for events. Zigbee-ad internally creates a group with an ID that's equal to the device ID.
+
+* `association_info` is an object describing a specific association group:
+
+| Field        | Type   | Example                                   | Description                                                       |
+|--------------|--------|-------------------------------------------|-------------------------------------------------------------------|
+| `group`      | int    | `1`, `3`                                  | Id of the supported group.                                        |
+| `trigger`    | string | `"Sensor - Temperature"`                  | General description of the trigger type.                          |
+| `name`       | string | `"Lifeline"`, `"Dimmer (S1)"`             | Name of the group, specified by the device manufacturer.          |
+| `commands`   | object | `[{"class": 128, "command": 3}]`          | Array of [`commands`](#definitions).                              |
+
+* `commands` is an int_map describing specific commands sent by an association:
+ 
+| Field        | Type | Example      | Description                                           |
+|--------------|------|--------------|-------------------------------------------------------|
+| `class`      | int  | `104`, `128` | Command class.                                        |
```

Comment:
@PaulinaDrozda Once again, we have class which is ZWave and in ZIgbee theese are clusters. @PaulinaDrozda please align with @BudzikM.